src/ui/game/GameScreen.js

```diff
@@ -225,9 +225,12 @@ export default class GameScreen extends Component {
                   id: 0,
                   messages: [],
                   status: 'created',
+                  draft: '',
                   chatsDisabled: !game.tree
                 }}
-                onSubmit={this._onChat} />
+                onSubmit={this._onChat}
+                onDraft={()=>{}}
```

Comment:
see previous comment about making `onDraft` optional, in which case this can be left out